### PR TITLE
Kumarde/rdn attributes

### PR DESCRIPTION
**Pull request by kumarde**
Adding check for multiple RDN Attributes. This addresses #31.

### CODE REVIEW

File: lints/base.go [hunk 1]
```diff
@@ -12,7 +12,7 @@ var (
 
 type lintReportUpdater func(*LintReport, ResultStruct)
 
-const ZLintVersion = 2
+const ZLintVersion = 3
```
Comment (dadrian) [inline lints/base.go]: Do we update this on new lints, or when we make breaking changes?

File: lints/lint_issuer_multiple_rdn.go [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+import (
+	"encoding/asn1"
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zcrypto/x509/pkix"
+	"github.com/zmap/zlint/util"
+)
+
+type IssuerRDNHasMultipleAttribute struct {
+	// Internal data here
```
Comment (dadrian) [inline lints/lint_issuer_multiple_rdn.go]: Just let this be an empty struct, no comment needed.

`type IssuerRDNHasMultipleAttribute struct {}`

File: lints/lint_issuer_multiple_rdn.go [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+func (l *IssuerRDNHasMultipleAttribute) Initialize() error {
+	return nil
+}
+
+func (l *IssuerRDNHasMultipleAttribute) CheckApplies(c *x509.Certificate) bool {
```
Comment (dadrian) [inline lints/lint_issuer_multiple_rdn.go]: Does this really apply to all certificates and not just subscriber certificates?

File: lints/lint_subject_multiple_rdn.go [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+import (
+	"encoding/asn1"
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zcrypto/x509/pkix"
+	"github.com/zmap/zlint/util"
+)
+
+type SubjectRDNHasMultipleAttribute struct {
+	// Internal data here
```
Comment (dadrian) [inline lints/lint_subject_multiple_rdn.go]: Ditto

File: lints/lint_subject_multiple_rdn.go [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+func (l *SubjectRDNHasMultipleAttribute) Initialize() error {
+	return nil
+}
+
+func (l *SubjectRDNHasMultipleAttribute) CheckApplies(c *x509.Certificate) bool {
+	return true
```
Comment (dadrian) [inline lints/lint_subject_multiple_rdn.go]: Ditto

File: testlint/testCerts/issuerRDNTwoAttribute.pem [hunk 6]
```diff
@@ -0,0 +1,16 @@
+-----BEGIN CERTIFICATE-----
```
Comment (dadrian) [inline testlint/testCerts/issuerRDNTwoAttribute.pem]: I don't know if we've been doing this in the past, but I like to have the OpenSSL -text output either prepended or appended to the PEM for test certificates, so they're a little easier to understand when you go look at them later.

```console
$ openssl x509 -in issuerRDNTwoAttribute.pem -out temp.pem -text
$ mv temp.pem issuerRDNTwoAttribute.pem
```

File: lints/base.go (hunk 1 above)
Comment (kumarde) [inline lints/base.go]: Based on a conversation w/ Zakir, he mentioned doing this for new lints. Maybe it's something we need to think about more closely.

File: lints/lint_issuer_multiple_rdn.go (hunk 2 above)
Comment (kumarde) [inline lints/lint_issuer_multiple_rdn.go]: Done

File: lints/lint_issuer_multiple_rdn.go (hunk 3 above)
Comment (kumarde) [inline lints/lint_issuer_multiple_rdn.go]: This is from AWS Certlint, and it looks like they check this on every certificate. But I can't find it in an RFC or the BR, so unfortunately I don't have the text to support it

File: lints/lint_subject_multiple_rdn.go (hunk 4 above)
Comment (kumarde) [inline lints/lint_subject_multiple_rdn.go]: done

File: lints/lint_subject_multiple_rdn.go (hunk 5 above)
Comment (kumarde) [inline lints/lint_subject_multiple_rdn.go]: done

File: lints/lint_issuer_multiple_rdn.go (hunk 3 above)
Comment (kumarde) [inline lints/lint_issuer_multiple_rdn.go]: done

File: testlint/testCerts/issuerRDNTwoAttribute.pem (hunk 6 above)
Comment (kumarde) [inline testlint/testCerts/issuerRDNTwoAttribute.pem]: we haven't been doing this, but it's a good idea. I'll send a PR where I do this for all our test certs.

**Pull request merged**